test/unittests/testData/ngsi10.restReply.updateContextResponse01.ok.valid.xml

```diff
@@ -2,5 +2,6 @@
   <errorCode>
     <code>404</code>
     <reasonPhrase>No context element found</reasonPhrase>
+    <details>detail</details>
```

Comment:
That means a "slight" functional change? I think is ok (it's very minor and it nos "violating" the XSD, so although clients should be prepared for this newly introduced details tag), but just to be sure...